x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/manual_configuration.tsx

```diff
@@ -0,0 +1,319 @@
+                      )}
+                    </EuiButtonEmpty>
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiButton
+                      data-test-subj="enterpriseSearchFlyoutManualConfigContentSaveButton"
+                      onClick={() => {
+                        createConnector({
+                          isSelfManaged: selfManagePreference === 'selfManaged',
+                          shouldGenerateAfterCreate: false,
+                          shouldNavigateToConnectorAfterCreate: true,
+                        });
+                      }}
+                      fill
+                    >
+                      {i18n.translate(
+                        'xpack.enterpriseSearch.createConnector.flyoutManualConfigContent.saveConfigurationButtonLabel',
+                        { defaultMessage: 'Save configuration' }
+                      )}
+                    </EuiButton>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </EuiFlyoutFooter>
+            </>
+          )}
+          {flyoutContent === 'client' && (
+            <>
+              <EuiFlyoutHeader hasBorder>
+                <EuiTitle size="m">
+                  <h2 id={simpleFlyoutTitleId}>
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.createConnector.manualConfiguration.h2.cliLabel',
+                      {
+                        defaultMessage: 'CLI',
+                      }
+                    )}
+                  </h2>
+                </EuiTitle>
+              </EuiFlyoutHeader>
+              <EuiFlyoutBody>
+                <EuiText size="s">
+                  <p>
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.createConnector.manualConfiguration.p.youCanAlsoUseLabel',
+                      {
+                        defaultMessage: 'You can also use the',
+                      }
+                    )}{' '}
+                    <EuiLink
+                      data-test-subj="enterpriseSearchManualConfigurationConnectorsCliLink"
+                      href="https://github.com/elastic/connectors/blob/main/docs/CLI.md"
+                      target="_blank"
+                      external
+                    >
+                      {i18n.translate(
+                        'xpack.enterpriseSearch.createConnector.manualConfiguration.connectorsCLILinkLabel',
+                        {
+                          defaultMessage: 'connectors CLI',
+                        }
+                      )}
+                    </EuiLink>{' '}
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.createConnector.manualConfiguration.p.commandlineInterfaceTheFollowingLabel',
+                      {
+                        defaultMessage:
+                          'command-line interface. The following command creates a new connector attached to the',
```

Comment:
this ends up reading `connectors CLI command-line interface.` which is redundant. CLI means command-line interface.